src/main/java/appeng/parts/automation/PartFormationPlane.java

```diff
@@ -409,26 +222,6 @@ private boolean isTransitionPlane( final TileEntity blockTileEntity, final AEPar
 		return Collections.emptyList();
 	}
 
-	@Override
-	public int getPriority()
-	{
-		return this.priority;
-	}
-
-	@Override
-	public void setPriority( final int newValue )
-	{
-		this.priority = newValue;
-		this.getHost().markForSave();
-		this.updateHandler();
-	}
-
-	@Override
-	public void blinkCell( final int slot )
-	{
-		// :P
-	}
-
 	@Override
 	public IAEItemStack injectItems( final IAEItemStack input, final Actionable type, final IActionSource src )
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') The Cyclomatic Complexity of this method "injectItems" is 35 which is greater than 10 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)